### PR TITLE
DDF-2064: Investigate and fix DDF itest Windows CI failures

### PR DESCRIPTION
**Pull request by figliold**
#### What does this PR do?
Fixes DDF itest Windows CI failures
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@garrettfreibott @Lambeaux @oconnormi @bantillo 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@lessarderic
@shaundmorris

#### What are the relevant tickets?
DDF-2064
#### Checklist:

- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

 - Changed code to make Karaf unpack directory configurable
 - Added waitForAllBundles() after search UI feature is started in TestSingleSignOn.java

### CODE REVIEW

**Comment by garrettfreibott** [comment]
LGTM 👍

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java [hunk 1]
```diff
@@ -373,13 +373,15 @@ protected UrlResourceReaderConfigurator getUrlResourceReaderConfigurator() {
     }
 
     protected Option[] configureDistribution() {
+        String unpackDirectory = System.getProperty("karaf.unpack.directory");
+        unpackDirectory = (unpackDirectory == null) ? "target/exam" : unpackDirectory;
```
Comment (lessarderic) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java]: Should we have a constant for `target/exam`?

Comment (lessarderic) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java]: You know what. Let's wait and see if the fix I have for the Bamboo build works and if it does, we probably should just remove this whole thing.

**Comment by bantillo** [comment]
PRBS passed.

**Pull request merged**